### PR TITLE
EFR32 : Wifi Code movement from SDK support repo to CSA

### PR DESCRIPTION
**Pull request by sharad-patil24**
EFR32 Wifi implementation was present in SDK Support repo ( https://github.com/SiliconLabs/sdk_support ).
The Implementation is moved to ( src/platform/EFR32 and example/platform/efr32) folder to meet the standards.

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

**Comment by sharad-patil24** [comment]
Waiting for 1 more Approval for  PR.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**
